### PR TITLE
Vue3 initial work

### PR DESCRIPTION
**Pull request by james090500**
### Description:
Some initial work to support Vue3

### Issue Reference
#158

### CODE REVIEW

File: packages/demo/package.json [hunk 1]
```diff
@@ -1,48 +1,46 @@
 {
   "name": "demo",
-	"version": "1.10.1",
+  "version": "1.11",
```
Comment (mazipan) [inline packages/demo/package.json:3]:
```suggestion
  "version": "2.0.0",
```

**Comment by airnichols13** [comment]
Any update on getting this update in? I'm also in the process of updating my site to vue 3 and would be great if this package could work with it!

**Comment by james090500** [comment]
Anything else needed before we merge @mazipan ?

**Comment by andreazevedobauru** [comment]
I recently developed a site with vue 3, it would be great if this feature was implemented.
Is there any prediction for this?
Thanks

**Pull request merged**